syntax = "proto3";

package me.eax.examples.game;

message SlideMeanFilter {
    int32 k = 1;
    bool need_restart = 2;
    int32 window = 3;
    int32 mean_SUM = 4;
    repeated int32 buf = 5[packed = true];
}